### PR TITLE
Ea 2462 schemaorg entity

### PR DESCRIPTION
**Pull request by SrishtiSingh-eu**

### CODE REVIEW

File: corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java [hunk 1]
```diff
@@ -81,7 +82,6 @@ public void toSchemaOrgEdmOrganizationTest() throws IOException {
         JsonLdSerializer serializer = new JsonLdSerializer();
         output = serializer.serialize(thingObject);
         Assert.assertNotNull(output);
-
         try (InputStream stream = getClass().getResourceAsStream(MockBeanConstants.EDMORGANIZATION_FILE)) {
```
Comment (abidaJustice) [inline corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java:85]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.

Comment (Luthien-in-edhil) [reply corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java:85]: @SrishtiSingh-eu do you know who Abida Justice is?

Comment (SrishtiSingh-eu) [reply corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java:85]: @Luthien-in-edhil no i don't know. And how she has access to review?

Comment (SrishtiSingh-eu) [reply corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java:85]: shouldn't this branch be merged already ?

Comment (Luthien-in-edhil) [reply corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java:85]: Just had a look, but Abida posts the exact same comment on dozens of repositories. Looks like a confused person to me.

**Pull request merged**